pkg/application/settings.go

```diff
@@ -1,16 +1,21 @@
 package application
 
 import (
-	"github.com/TheQueenIsDead/budge/pkg/database/models"
 	"github.com/labstack/echo/v4"
 	"net/http"
 )
 
 func (app *Application) Settings(c echo.Context) error {
 
-	config := app.integrations.Config()
+	akahuConfig, err := app.store.GetAkahuSettings()
+	if err != nil {
+		app.Toast(c, "Error", "Could not get Akahu settings.")
+		return c.NoContent(http.StatusInternalServerError)
+	}
 	return c.Render(http.StatusOK, "settings", map[string]interface{}{
-		"config": config,
+		"akahuAppToken":  akahuConfig.AppToken,
+		"akahuUserToken": akahuConfig.UserToken,
+		"akahuLastSync":  akahuConfig.LastSync,
```

Comment:
Moved all the config to the top level, as previously we were displaying the integration config as opposed to the stuff stored in the DB, which we may add to in future.

Tried doing a few layers of maps and interfaces, but the frontend has a hard time calling functions when it's too nested, this suffices